support/jakarta-ee/src/main/java/org/apache/shiro/ee/listeners/IniEnvironment.java

```diff
@@ -44,7 +47,9 @@ public class IniEnvironment extends IniWebEnvironment {
     private String otherConfigLocation;
 
     @SuppressWarnings("deprecation")
+    @RequiredArgsConstructor
```

Comment:
nit, is this lombok annotation needed, it looks like all of the constructors are defined?

Reply:
Yes. The final variable is initialized by the annotation otherwise it’ll fall

Reply:
Ahh, I see it now, sorry. 🤦 
Inner class + side-by-side diff, i missed it